Yoast/javascript
# Implement Hungarian passive voice

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements the Hungarian passive voice.
* [Yoast SEO Free] Adds the assessment for detecting passive voice in Hungarian.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Set your website to Hungarian.
* Test this functionality in Yoast SEO Free.

* Create a new empty post and add these sentences to your post:
`A csomag kiszállításra került.`
`A lakás ára kifizetésre került.`
`A levél 100 éve íródott.`
`A telefon gyorsan töltődik.`
`A ruha fel fog hajtódni.`
`A kád meg fog betöltődni vízzel.`
`A függöny ki van mosva.`
`Az ajó festve van.`
`Mázolva van a kerítés.`

* Check that the passive voice bullet turns red and all sentences are highlighted.

* Add this sentence and check that it does not get highlighted:
`A mosómedve vadászik az erdőben.`

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-527
